dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -6,7 +6,7 @@
 from unittest import mock
```

Comment:
general question about the tests for your implementation. Do you have test cases that cover where `df_series.to_numpy()` would be called? For example:
```python
if df_series.is_empty():
    num_negatives_value = 0
elif self._greater_than_64_bit:
    num_negatives_value = int((df_series.to_numpy() < 0).sum())
else:
    num_negatives_value = int((df_series < 0).sum())
```
do you have test cases that will go in both the `elif` and the `else` here?

Reply:
yup these are covered by the int_column_profiler when it tests values of 64 bits.